src/content/concepts/loaders.md

```diff
@@ -80,6 +81,7 @@ module.exports = {
 };
 ```
 
+> Note: For detailed information about [style-loader](/loaders/style-loader), [css-loader](/loaders/css-loader) and [sass-loader](/loaders/sass-loader), please visit the links.
```

Comment:
Move them to the further reading section

Reply:
@EugeneHlushko where is that section?

Reply:
Its here: https://webpack.js.org/contribute/writers-guide#yaml-frontmatter

Reply:
@EugeneHlushko thanks

Reply:
@EugeneHlushko done with the further reading section